2018-04-25 21:24:51 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.panic(MVStore.java:378)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:361)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2930)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2476)
	at org.h2.engine.Database.open(Database.java:697)
	at org.h2.engine.Database.openDatabase(Database.java:276)
	at org.h2.engine.Database.<init>(Database.java:270)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:354)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:100)
	at org.h2.Driver.connect(Driver.java:69)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:375)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:204)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:459)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:533)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:114)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:82)
	at hikari_cp.core$make_datasource.invokeStatic(core.clj:249)
	at hikari_cp.core$make_datasource.invoke(core.clj:246)
	at conman.core$connect_BANG_.invokeStatic(core.clj:126)
	at conman.core$connect_BANG_.invoke(core.clj:122)
	at cwbn.db.core$eval30459$fn__30460.invoke(core.clj:9)
	at mount.core$record_BANG_.invokeStatic(core.cljc:86)
	at mount.core$record_BANG_.invoke(core.cljc:85)
	at mount.core$up$fn__273.invoke(core.cljc:93)
	at mount.core$up.invokeStatic(core.cljc:92)
	at mount.core$up.invoke(core.cljc:90)
	at mount.core$bring.invokeStatic(core.cljc:242)
	at mount.core$bring.invoke(core.cljc:234)
	at mount.core$start.invokeStatic(core.cljc:284)
	at mount.core$start.doInvoke(core.cljc:276)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at mount.core$start_with_args.invokeStatic(core.cljc:384)
	at mount.core$start_with_args.doInvoke(core.cljc:380)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at cwbn.core$start_app.invokeStatic(core.clj:41)
	at cwbn.core$start_app.invoke(core.clj:40)
	at cwbn.core$_main.invokeStatic(core.clj:64)
	at cwbn.core$_main.doInvoke(core.clj:48)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.Var.invoke(Var.java:377)
	at user$eval34586.invokeStatic(form-init2412724271599853233.clj:1)
	at user$eval34586.invoke(form-init2412724271599853233.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:7062)
	at clojure.lang.Compiler.eval(Compiler.java:7052)
	at clojure.lang.Compiler.load(Compiler.java:7514)
	at clojure.lang.Compiler.loadFile(Compiler.java:7452)
	at clojure.main$load_script.invokeStatic(main.clj:278)
	at clojure.main$init_opt.invokeStatic(main.clj:280)
	at clojure.main$init_opt.invoke(main.clj:280)
	at clojure.main$initialize.invokeStatic(main.clj:311)
	at clojure.main$null_opt.invokeStatic(main.clj:345)
	at clojure.main$null_opt.invoke(main.clj:342)
	at clojure.main$main.invokeStatic(main.clj:424)
	at clojure.main$main.doInvoke(main.clj:387)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.lang.Var.applyTo(Var.java:702)
	at clojure.main.main(main.java:37)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 69 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:765)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:347)
	... 64 more
2018-04-25 21:26:06 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.panic(MVStore.java:378)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:361)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2930)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2476)
	at org.h2.engine.Database.open(Database.java:697)
	at org.h2.engine.Database.openDatabase(Database.java:276)
	at org.h2.engine.Database.<init>(Database.java:270)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:354)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:100)
	at org.h2.Driver.connect(Driver.java:69)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:375)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:204)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:459)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:533)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:114)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:82)
	at hikari_cp.core$make_datasource.invokeStatic(core.clj:249)
	at hikari_cp.core$make_datasource.invoke(core.clj:246)
	at conman.core$connect_BANG_.invokeStatic(core.clj:126)
	at conman.core$connect_BANG_.invoke(core.clj:122)
	at cwbn.db.core$eval30459$fn__30460.invoke(core.clj:9)
	at mount.core$record_BANG_.invokeStatic(core.cljc:86)
	at mount.core$record_BANG_.invoke(core.cljc:85)
	at mount.core$up$fn__273.invoke(core.cljc:93)
	at mount.core$up.invokeStatic(core.cljc:92)
	at mount.core$up.invoke(core.cljc:90)
	at mount.core$bring.invokeStatic(core.cljc:242)
	at mount.core$bring.invoke(core.cljc:234)
	at mount.core$start.invokeStatic(core.cljc:284)
	at mount.core$start.doInvoke(core.cljc:276)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at mount.core$start_with_args.invokeStatic(core.cljc:384)
	at mount.core$start_with_args.doInvoke(core.cljc:380)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at cwbn.core$start_app.invokeStatic(core.clj:41)
	at cwbn.core$start_app.invoke(core.clj:40)
	at cwbn.core$_main.invokeStatic(core.clj:64)
	at cwbn.core$_main.doInvoke(core.clj:48)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.Var.invoke(Var.java:377)
	at user$eval34586.invokeStatic(form-init2831077149980880805.clj:1)
	at user$eval34586.invoke(form-init2831077149980880805.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:7062)
	at clojure.lang.Compiler.eval(Compiler.java:7052)
	at clojure.lang.Compiler.load(Compiler.java:7514)
	at clojure.lang.Compiler.loadFile(Compiler.java:7452)
	at clojure.main$load_script.invokeStatic(main.clj:278)
	at clojure.main$init_opt.invokeStatic(main.clj:280)
	at clojure.main$init_opt.invoke(main.clj:280)
	at clojure.main$initialize.invokeStatic(main.clj:311)
	at clojure.main$null_opt.invokeStatic(main.clj:345)
	at clojure.main$null_opt.invoke(main.clj:342)
	at clojure.main$main.invokeStatic(main.clj:424)
	at clojure.main$main.doInvoke(main.clj:387)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.lang.Var.applyTo(Var.java:702)
	at clojure.main.main(main.java:37)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 69 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:765)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:347)
	... 64 more
2018-04-25 21:28:19 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.panic(MVStore.java:378)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:361)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2930)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2476)
	at org.h2.engine.Database.open(Database.java:697)
	at org.h2.engine.Database.openDatabase(Database.java:276)
	at org.h2.engine.Database.<init>(Database.java:270)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:354)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:100)
	at org.h2.Driver.connect(Driver.java:69)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:375)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:204)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:459)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:533)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:114)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:82)
	at hikari_cp.core$make_datasource.invokeStatic(core.clj:249)
	at hikari_cp.core$make_datasource.invoke(core.clj:246)
	at conman.core$connect_BANG_.invokeStatic(core.clj:126)
	at conman.core$connect_BANG_.invoke(core.clj:122)
	at cwbn.db.core$eval30459$fn__30460.invoke(core.clj:9)
	at mount.core$record_BANG_.invokeStatic(core.cljc:86)
	at mount.core$record_BANG_.invoke(core.cljc:85)
	at mount.core$up$fn__273.invoke(core.cljc:93)
	at mount.core$up.invokeStatic(core.cljc:92)
	at mount.core$up.invoke(core.cljc:90)
	at mount.core$bring.invokeStatic(core.cljc:242)
	at mount.core$bring.invoke(core.cljc:234)
	at mount.core$start.invokeStatic(core.cljc:284)
	at mount.core$start.doInvoke(core.cljc:276)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at mount.core$start_with_args.invokeStatic(core.cljc:384)
	at mount.core$start_with_args.doInvoke(core.cljc:380)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at cwbn.core$start_app.invokeStatic(core.clj:41)
	at cwbn.core$start_app.invoke(core.clj:40)
	at cwbn.core$_main.invokeStatic(core.clj:64)
	at cwbn.core$_main.doInvoke(core.clj:48)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.Var.invoke(Var.java:377)
	at user$eval34586.invokeStatic(form-init6294631264872130742.clj:1)
	at user$eval34586.invoke(form-init6294631264872130742.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:7062)
	at clojure.lang.Compiler.eval(Compiler.java:7052)
	at clojure.lang.Compiler.load(Compiler.java:7514)
	at clojure.lang.Compiler.loadFile(Compiler.java:7452)
	at clojure.main$load_script.invokeStatic(main.clj:278)
	at clojure.main$init_opt.invokeStatic(main.clj:280)
	at clojure.main$init_opt.invoke(main.clj:280)
	at clojure.main$initialize.invokeStatic(main.clj:311)
	at clojure.main$null_opt.invokeStatic(main.clj:345)
	at clojure.main$null_opt.invoke(main.clj:342)
	at clojure.main$main.invokeStatic(main.clj:424)
	at clojure.main$main.doInvoke(main.clj:387)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.lang.Var.applyTo(Var.java:702)
	at clojure.main.main(main.java:37)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 69 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:765)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:347)
	... 64 more
2018-04-26 19:42:47 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.panic(MVStore.java:378)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:361)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2930)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2476)
	at org.h2.engine.Database.open(Database.java:697)
	at org.h2.engine.Database.openDatabase(Database.java:276)
	at org.h2.engine.Database.<init>(Database.java:270)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:354)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:100)
	at org.h2.Driver.connect(Driver.java:69)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:375)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:204)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:459)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:533)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:114)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:82)
	at hikari_cp.core$make_datasource.invokeStatic(core.clj:249)
	at hikari_cp.core$make_datasource.invoke(core.clj:246)
	at conman.core$connect_BANG_.invokeStatic(core.clj:126)
	at conman.core$connect_BANG_.invoke(core.clj:122)
	at cwbn.db.core$eval30459$fn__30460.invoke(core.clj:9)
	at mount.core$record_BANG_.invokeStatic(core.cljc:86)
	at mount.core$record_BANG_.invoke(core.cljc:85)
	at mount.core$up$fn__273.invoke(core.cljc:93)
	at mount.core$up.invokeStatic(core.cljc:92)
	at mount.core$up.invoke(core.cljc:90)
	at mount.core$bring.invokeStatic(core.cljc:242)
	at mount.core$bring.invoke(core.cljc:234)
	at mount.core$start.invokeStatic(core.cljc:284)
	at mount.core$start.doInvoke(core.cljc:276)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at mount.core$start_with_args.invokeStatic(core.cljc:384)
	at mount.core$start_with_args.doInvoke(core.cljc:380)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at cwbn.core$start_app.invokeStatic(core.clj:41)
	at cwbn.core$start_app.invoke(core.clj:40)
	at cwbn.core$_main.invokeStatic(core.clj:64)
	at cwbn.core$_main.doInvoke(core.clj:48)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.Var.invoke(Var.java:377)
	at user$eval34586.invokeStatic(form-init9006695997303855286.clj:1)
	at user$eval34586.invoke(form-init9006695997303855286.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:7062)
	at clojure.lang.Compiler.eval(Compiler.java:7052)
	at clojure.lang.Compiler.load(Compiler.java:7514)
	at clojure.lang.Compiler.loadFile(Compiler.java:7452)
	at clojure.main$load_script.invokeStatic(main.clj:278)
	at clojure.main$init_opt.invokeStatic(main.clj:280)
	at clojure.main$init_opt.invoke(main.clj:280)
	at clojure.main$initialize.invokeStatic(main.clj:311)
	at clojure.main$null_opt.invokeStatic(main.clj:345)
	at clojure.main$null_opt.invoke(main.clj:342)
	at clojure.main$main.invokeStatic(main.clj:424)
	at clojure.main$main.doInvoke(main.clj:387)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.lang.Var.applyTo(Var.java:702)
	at clojure.main.main(main.java:37)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 69 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:765)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:347)
	... 64 more
2018-04-26 20:17:11 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.panic(MVStore.java:378)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:361)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2930)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2476)
	at org.h2.engine.Database.open(Database.java:697)
	at org.h2.engine.Database.openDatabase(Database.java:276)
	at org.h2.engine.Database.<init>(Database.java:270)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:354)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:100)
	at org.h2.Driver.connect(Driver.java:69)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:375)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:204)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:459)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:533)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:114)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:82)
	at hikari_cp.core$make_datasource.invokeStatic(core.clj:249)
	at hikari_cp.core$make_datasource.invoke(core.clj:246)
	at conman.core$connect_BANG_.invokeStatic(core.clj:126)
	at conman.core$connect_BANG_.invoke(core.clj:122)
	at cwbn.db.core$eval30459$fn__30460.invoke(core.clj:9)
	at mount.core$record_BANG_.invokeStatic(core.cljc:86)
	at mount.core$record_BANG_.invoke(core.cljc:85)
	at mount.core$up$fn__273.invoke(core.cljc:93)
	at mount.core$up.invokeStatic(core.cljc:92)
	at mount.core$up.invoke(core.cljc:90)
	at mount.core$bring.invokeStatic(core.cljc:242)
	at mount.core$bring.invoke(core.cljc:234)
	at mount.core$start.invokeStatic(core.cljc:284)
	at mount.core$start.doInvoke(core.cljc:276)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at mount.core$start_with_args.invokeStatic(core.cljc:384)
	at mount.core$start_with_args.doInvoke(core.cljc:380)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at cwbn.core$start_app.invokeStatic(core.clj:41)
	at cwbn.core$start_app.invoke(core.clj:40)
	at cwbn.core$_main.invokeStatic(core.clj:64)
	at cwbn.core$_main.doInvoke(core.clj:48)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.Var.invoke(Var.java:377)
	at user$eval34586.invokeStatic(form-init6121940999009628993.clj:1)
	at user$eval34586.invoke(form-init6121940999009628993.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:7062)
	at clojure.lang.Compiler.eval(Compiler.java:7052)
	at clojure.lang.Compiler.load(Compiler.java:7514)
	at clojure.lang.Compiler.loadFile(Compiler.java:7452)
	at clojure.main$load_script.invokeStatic(main.clj:278)
	at clojure.main$init_opt.invokeStatic(main.clj:280)
	at clojure.main$init_opt.invoke(main.clj:280)
	at clojure.main$initialize.invokeStatic(main.clj:311)
	at clojure.main$null_opt.invokeStatic(main.clj:345)
	at clojure.main$null_opt.invoke(main.clj:342)
	at clojure.main$main.invokeStatic(main.clj:424)
	at clojure.main$main.doInvoke(main.clj:387)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.lang.Var.applyTo(Var.java:702)
	at clojure.main.main(main.java:37)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 69 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:765)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:347)
	... 64 more
2018-04-26 20:19:01 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.panic(MVStore.java:378)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:361)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2930)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2476)
	at org.h2.engine.Database.open(Database.java:697)
	at org.h2.engine.Database.openDatabase(Database.java:276)
	at org.h2.engine.Database.<init>(Database.java:270)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:354)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:100)
	at org.h2.Driver.connect(Driver.java:69)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:375)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:204)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:459)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:533)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:114)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:82)
	at hikari_cp.core$make_datasource.invokeStatic(core.clj:249)
	at hikari_cp.core$make_datasource.invoke(core.clj:246)
	at conman.core$connect_BANG_.invokeStatic(core.clj:126)
	at conman.core$connect_BANG_.invoke(core.clj:122)
	at cwbn.db.core$eval30459$fn__30460.invoke(core.clj:9)
	at mount.core$record_BANG_.invokeStatic(core.cljc:86)
	at mount.core$record_BANG_.invoke(core.cljc:85)
	at mount.core$up$fn__273.invoke(core.cljc:93)
	at mount.core$up.invokeStatic(core.cljc:92)
	at mount.core$up.invoke(core.cljc:90)
	at mount.core$bring.invokeStatic(core.cljc:242)
	at mount.core$bring.invoke(core.cljc:234)
	at mount.core$start.invokeStatic(core.cljc:284)
	at mount.core$start.doInvoke(core.cljc:276)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at mount.core$start_with_args.invokeStatic(core.cljc:384)
	at mount.core$start_with_args.doInvoke(core.cljc:380)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at cwbn.core$start_app.invokeStatic(core.clj:41)
	at cwbn.core$start_app.invoke(core.clj:40)
	at cwbn.core$_main.invokeStatic(core.clj:64)
	at cwbn.core$_main.doInvoke(core.clj:48)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.Var.invoke(Var.java:377)
	at user$eval34586.invokeStatic(form-init5728920741536664158.clj:1)
	at user$eval34586.invoke(form-init5728920741536664158.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:7062)
	at clojure.lang.Compiler.eval(Compiler.java:7052)
	at clojure.lang.Compiler.load(Compiler.java:7514)
	at clojure.lang.Compiler.loadFile(Compiler.java:7452)
	at clojure.main$load_script.invokeStatic(main.clj:278)
	at clojure.main$init_opt.invokeStatic(main.clj:280)
	at clojure.main$init_opt.invoke(main.clj:280)
	at clojure.main$initialize.invokeStatic(main.clj:311)
	at clojure.main$null_opt.invokeStatic(main.clj:345)
	at clojure.main$null_opt.invoke(main.clj:342)
	at clojure.main$main.invokeStatic(main.clj:424)
	at clojure.main$main.doInvoke(main.clj:387)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.lang.Var.applyTo(Var.java:702)
	at clojure.main.main(main.java:37)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 69 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:765)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:347)
	... 64 more
2018-04-26 20:21:04 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.panic(MVStore.java:378)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:361)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2930)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2476)
	at org.h2.engine.Database.open(Database.java:697)
	at org.h2.engine.Database.openDatabase(Database.java:276)
	at org.h2.engine.Database.<init>(Database.java:270)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:354)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:100)
	at org.h2.Driver.connect(Driver.java:69)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:375)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:204)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:459)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:533)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:114)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:82)
	at hikari_cp.core$make_datasource.invokeStatic(core.clj:249)
	at hikari_cp.core$make_datasource.invoke(core.clj:246)
	at conman.core$connect_BANG_.invokeStatic(core.clj:126)
	at conman.core$connect_BANG_.invoke(core.clj:122)
	at cwbn.db.core$eval30459$fn__30460.invoke(core.clj:9)
	at mount.core$record_BANG_.invokeStatic(core.cljc:86)
	at mount.core$record_BANG_.invoke(core.cljc:85)
	at mount.core$up$fn__273.invoke(core.cljc:93)
	at mount.core$up.invokeStatic(core.cljc:92)
	at mount.core$up.invoke(core.cljc:90)
	at mount.core$bring.invokeStatic(core.cljc:242)
	at mount.core$bring.invoke(core.cljc:234)
	at mount.core$start.invokeStatic(core.cljc:284)
	at mount.core$start.doInvoke(core.cljc:276)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at mount.core$start_with_args.invokeStatic(core.cljc:384)
	at mount.core$start_with_args.doInvoke(core.cljc:380)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at cwbn.core$start_app.invokeStatic(core.clj:41)
	at cwbn.core$start_app.invoke(core.clj:40)
	at cwbn.core$_main.invokeStatic(core.clj:64)
	at cwbn.core$_main.doInvoke(core.clj:48)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.Var.invoke(Var.java:377)
	at user$eval34586.invokeStatic(form-init5075701404780818320.clj:1)
	at user$eval34586.invoke(form-init5075701404780818320.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:7062)
	at clojure.lang.Compiler.eval(Compiler.java:7052)
	at clojure.lang.Compiler.load(Compiler.java:7514)
	at clojure.lang.Compiler.loadFile(Compiler.java:7452)
	at clojure.main$load_script.invokeStatic(main.clj:278)
	at clojure.main$init_opt.invokeStatic(main.clj:280)
	at clojure.main$init_opt.invoke(main.clj:280)
	at clojure.main$initialize.invokeStatic(main.clj:311)
	at clojure.main$null_opt.invokeStatic(main.clj:345)
	at clojure.main$null_opt.invoke(main.clj:342)
	at clojure.main$main.invokeStatic(main.clj:424)
	at clojure.main$main.doInvoke(main.clj:387)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.lang.Var.applyTo(Var.java:702)
	at clojure.main.main(main.java:37)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 69 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Projects/cwbn-clj/cwbn/cwbn_dev.db.mv.db [1.4.196/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:765)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:347)
	... 64 more
